lms/djangoapps/courseware/views/views.py

```diff
@@ -863,6 +863,92 @@ def post(self, request, course_id):
         return redirect(reverse('about_course', args=[text_type(course_key)]))
 
 
+@ensure_csrf_cookie
+@cache_if_anonymous()
+def course_page(request, course_id):
+    course_id_tokens = course_id.split('+')
+    org = course_id_tokens[0]
+    number = course_id_tokens[1]
```

Comment:
working with indexes can break the code, we need to validate the course IDs before splitting.

Reply:
this would never be the case, since org and number are mandatory fields while creating a course
besides there is a url validation in place for the course key